locales/lo/webextension.properties

```diff
@@ -1,14 +1,14 @@
 # Clip in this context refers to a portion of an image
 addonDescription = ຖ່າຍຄຣິບ ແລະ ພາບຫນ້າຈໍຈາກຫນ້າເວັບ ແລ້ວບັນທຶກໄວ້ຊົ່ວຄາວ ຫລື ຖາວອນ.
-addonAuthorsList = Mozilla <screenshots-feedback@mozilla.org>
+addonAuthorsList = Mozilla <screenshots-feedback@mozilla.com>
 # Also used for the title on the toolbar button
 contextMenuLabel = ຖ່າຍພາບຫນ້າຈໍ
 myShotsLink = ພາບຂອງຂ້ອຍ
 screenshotInstructions = ລາກ ຫລື ຄິກໃສ່ຫນ້າເວັບເພື່ອເລືອກເອົາບ່ອນທີ່ຕ້ອງການ. ກົດ ESC ເພື່ອຍົກເລີກ.
 saveScreenshotSelectedArea = ບັນທຶກ
 saveScreenshotVisibleArea = ບັນທຶກສ່ວນທີ່ເບິງເຫັນໄດ້
 saveScreenshotFullPage = ບັນທຶກຫມົດຫນ້າ
-cancelScreenshot = 
+cancelScreenshot =
```

Comment:
@flodolo I figured trimmed whitespace on the other strings was fine, but this one was blank.  still ok here?